main.rb

```diff
@@ -24,7 +25,9 @@ def main
     app.create_rental if selected.to_i == 5
     app.list_all_rentals_for_person_id if selected.to_i == 6
     puts 'Thank you for using this app!' if selected.to_i == 7
+    break if selected.to_i == 7
   end
```

Comment:
Great job team 🤝

- [ ] Kindly make sure the rental is created successfully without errors.
![create rental](https://user-images.githubusercontent.com/63917725/200689743-7f8e5000-7550-4b60-91d6-f12f9f1d0f46.PNG)

- [ ] You may also make sure the student is created successfully without hindering errors.
![create student](https://user-images.githubusercontent.com/63917725/200689898-7f5ddfa6-1d4b-4623-9fbd-87d3787c221a.PNG)

- [ ] Kindly make sure on exit from the student's app check the data is being saved (all the data: books, people, and rentals) into JSON files.